pydy/viz/scene.py

```diff
@@ -363,14 +363,10 @@ def generate_visualization_json_system(self, system, **kwargs):
                                           system.integrate(),
                                           system.constants.values(), **kwargs)
 
-    def create_static_html(self, overwrite=False, silent=False):
-
-        """Creates a directory named ``static`` in the current working
-        directory which contains all of the HTML, CSS, and Javascript files
-        required to run the visualization. Simply open ``static/index.html``
-        in a WebGL compliant browser to view and interact with the
-        visualization.
+    def create_static_html(self, copy=False, silent=False):
 
+        """
+        Copies json files into a static directory, pydy/viz/static by default,
```

Comment:
It does but only if ``copy=True`` is passed but by default it only copies into ``pydy/viz/static``